addons/web_editor/static/lib/odoo-editor/src/utils/sanitize.js

```diff
@@ -133,6 +133,11 @@ class Sanitize {
             }
         }
 
+        // Ensure unique ids on checklists.
+        if (node.nodeName == 'LI' && node.parentElement && node.parentElement.classList.contains('o_checklist')) {
+            node.setAttribute('id', Math.floor(new Date() * Math.random()));
```

Comment:
I'm not convinced by this random id. Why not simply always increment ?

Reply:
Because precisely it's not simple to always increment and avoid duplication errors. I was able to remove code that was specially dedicated to managing those ids (20 lines, just below this, triggered on every single change in the page), and reuse this for the stars.
Since the code was triggered on change, now that change is linked to an observer, if that code changes the dom it triggers the observer a second time. All in all ridiculously inefficient and complicated for something that can be handled in one line with a random id.

Reply:
Why not a uuid then ?